full-service/src/json_rpc/v1/api/response.rs

```diff
@@ -43,6 +43,7 @@ pub enum JsonCommandResponse {
     build_and_submit_transaction {
         transaction_log: TransactionLog,
         tx_proposal: TxProposal,
+        associated_txos: String,
```

Comment:
This likely needs to be part of `transaction_log`. The returned JSON looks something like this:
```
{
  "method": "build_and_submit_transaction",
  "result": {
    "transaction_log": {
      "id": "07d60c8fa81691cc0dcc5eb18e6b91c906e148d6306df5046bb96b281157d0aa",
      "account_id": "524917ec291ea181ac041be90dac475de0cae6afe9c915b9ed8d066edaf5d69e",
      "input_txos": [
        {
          "txo_id": "4bab82808685b52d1b5887550f8d63e43b7c809de34b19171ca8d48211a0cc31",
          "amount": {
            "value": "550996742999999",
            "token_id": "0"
          }
        }
      ],
      "output_txos": [
        {
          "txo_id_hex": "8f716e47f8aa720425519f237ca33f9d2e70380d0c844358af0fbe03da16c253",
          "amount": {
            "value": "100000",
            "token_id": "0"
          },
          "recipient_public_address_b58": "44WtVd7U1dxZ42NnwwyTocQ2Qqmghvcd3iPoSJAVcm8GjcvnkmYCUPHXYMQLC2KDMAYNEy6ixvJULi1eLmnKTEpN48afJW7SheFCRXrwfKG"
        }
      ],
      "change_txos": [
        {
          "txo_id_hex": "de948e9c56177119a9c6fc5324c3f650e760e24ce6d29063619091ed9fa50c27",
          "amount": {
            "value": "550996342899999",
            "token_id": "0"
          },
          "recipient_public_address_b58": "5yW6pV6iVD6xPgPnthxYGsH97wReoCGn5qCbbejZ9pvTEttXexcf4F9x8h42ptAhUXcVKusLWqmwLZ6PyDHxSbJtfQVt2H6pD5JwD1FeUvA"
        }
```
Each of these txo objects (that has the `txo_id_hex` field) should probably contain the txo public key.